packages/wdio-devtools-service/README.md

```diff
@@ -191,6 +290,28 @@ console.log(browser.getPageWeight())
 // }
 ```
 
+### Access Puppeteer Instance
+
+The service uses Puppeteer for its automation under the hood. You can get access to the used instance by calling the `getPuppeteer` command. __Note:__ Puppeteer commands are async and either needs to be called within the `call` command or handled via `async/await`:
+
+```js
+describe('use Puppeteer', () => {
+    it('by wrapping commands with call', () => {
+        browser.url('http://json.org')
+
+        const puppeteer = browser.getPuppeteer()
+        const page = browser.call(() => puppeteer.browser.pages())[0]
```

Comment:
why here you use `page` and in async example you use `pages`. I think it would be better to keep one flow in example